tools/slicec-cs/src/cs_compile.rs

```diff
@@ -81,30 +81,48 @@ mod test {
     #[test]
     fn compile_all_test_slice() {
```

Comment:
It seems that integrating this check as a test within the Rust code might not be the most appropriate approach. There isn't a specific unit of behavior that it's validating for `slicec-cs`. Rather, it appears better suited as a step within a GitHub action. The idea would be to invoke `slicec-cs` on all slice files, and if anything other than an exit code of 0 is returned, the action should fail. Thoughts, @externl?

Reply:
This test is primarily about helping to generating code coverage. So while it's not an AAA test it's still (IMO) quite useful.

Reply:
Just for context, it is _technically_ in the wrong place.
Integration tests like these _should_ be in a top level `tests` folder.
But we spent so much time writing unit tests, we just got used to putting the tests directly in the code,
even though by convention, sure, you should only do that for unit tests.

I don't have any vested stake in this code, but I do think that if we decide to do your suggestion, it should be in a separate PR than this one.